index.js

```diff
@@ -110,6 +111,19 @@ module.exports = function(config) {
         util.Formio.getToken = originalGetToken;
         util.Formio.Components.components.component.prototype.evalContext = originalEvalContext;
 
+        try {
+          if (config.maxOldSpace) {
+            const heap = process.memoryUsage().heapTotal / 1024 / 1024;
+
+            if ((config.maxOldSpace * 0.8) < heap) {
```

Comment:
This performs a garbage collection on every request? This seems like it would cause performance issues.

Reply:
We just check memory usage, I don't think, it really causes perfomance